extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js

```diff
@@ -26,13 +27,17 @@ define('ext.wikia.adEngine.provider.gpt.adSizeFilter', [
 			win.ads.context;
 	}
 
-	function removeUAPForFeaturedVideoPages(slotName, slotSizes) {
-		var adContext = getAdContext();
+	function removeFanTakeoverSizes(slotName, slotSizes) {
+		var adContext = getAdContext(),
+			recommendedVideoTestName = 'RECOMMENDED_VIDEO_AB',
+			runsRecommendedVideoABTest = abTest.getGroup(recommendedVideoTestName) && win.location.hostname.match(
+				/(dragonage|dragonball|elderscrolls|gta|memory-alpha|monsterhunter|naruto|marvelcinematicuniverse)((\.wikia\.com)|(\.wikia-dev\.pl))/g
```

Comment:
This regexp matches around 1156 domains:
```
SELECT city_domain FROM city_domains WHERE city_domain REGEXP '(dragonage|dragonball|elderscrolls|gta|memory-alpha|monsterhunter|naruto|marvelcinematicuniverse)((\.wikia\.com)|(\.wikia-dev\.pl))';

1156 rows in set (12.86 sec)
```
It would be much more safer to add wgFlag to DefaultSettings and set true value on selected 8 wikis. Then you can pass it to frontend through:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/AdEngine2ContextService.class.php#L78
and get it using `ext.wikia.adEngine.adContext`:
```
adContext.get('opts.isRecommendedVideoEnabled')
```